whynotearth/BrowTricks
# 341 change tenant signup

# Issue Being Addressed

#341 

# Type of PR


[ ] Bug Fix
[ ] Refactor
[ ] New Feature
[x] Update to Existing Feature
[ ] Other (state below)

# Description

**For Feature Updates:** 

Update tenant creation form (remove multi-step form)

# How to Test/Reproduce

Sign up (or login to an account that has no tenant), verify, you'll see the new tenant form.

# Screenshots/casts

![localhost_8080_signup](https://user-images.githubusercontent.com/510242/94729799-17c56e00-036f-11eb-81e6-eef223dbd09d.png)


# Additional Context

+ Fix a minor bug in formGeneralUtils.js error handler.

WIP: Still has a backend error on submit.
